java/pubsub/src/main/java/com/example/pubsub/CreatePullSubscriptionExample.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 Google Inc.
```

Comment:
Do we want to keep the "Google Inc." license header here? Or should we use OpenCensus header instead?

Reply:
I think we should keep it as-is since I'm copying the files - prefer to have it clearly attributed to "Google Inc." rather than having it look like we're claiming authorship.

I notice that many of the files in this repo (e.g. the ones under interoptest) are attributed to 'Google LLC' rather than 'OpenCensus Authors', so we may want to consider updating those.